### DISCUSSION TITLE
[GG] console needs a dedicated gg.txt file on SD

### DISCUSSION THREAD

**Discussion by PsyK0p4T** [sanni/cartreader#429] [Coding Corner]
Just made a GG adapter and I can confirm that ROM dumping process is running all fine. Expected hashes are here.
There is just an issue at the last step, at CRC32 search / file renaming : "sms.txt" file is used for SMS-raphnet, SMS-retrode, and also GG-retrode. Of course there is no Game Gear games or hashes in it. The console would need a dedicated file, same format as SMS one.

**Comment by nsx0r** [comment]
there you go:
[gg.txt](https://github.com/sanni/cartreader/files/9174753/gg.txt)

**Comment by PsyK0p4T** [comment]
Thanks a lot for the file nsx0r ! Just modified it to deal with encoding, and added its use to sms.ino
Tested again raphnet and retrode modes, everything working :)
